tests/args.test.js

```diff
@@ -40,9 +39,8 @@ describe("utils", () => {
     });
 
     it("mochitest output", () => {
-      expect(_getArgs("-i file.log")).toEqual(
+      expect(_getArgs("--read file.log")).toEqual(
         paramDefaults({
-          i: "file.log",
```

Comment:
`i` and `p` are so hard to remember. I killed them because for such a small CLI it's impossible to remember what they do